src/components/application_manager/rpc_plugins/sdl_rpc_plugin/test/commands/hmi/hmi_notifications_test.cc

```diff
@@ -1794,24 +1794,36 @@ TEST_F(HMICommandsNotificationsTest, OnDriverDistractionNotificationEmptyData) {
       CreateCommand<hmi::OnDriverDistractionNotification>(message);
 
   EXPECT_CALL(app_mngr_, set_driver_distraction_state(state));
-  EXPECT_CALL(app_mngr_, applications()).WillOnce(Return(applications_));
+  ON_CALL(app_mngr_, GetPolicyHandler())
```

Comment:
@LuxoftAKutsan I would suggest you to extract these lines to a separate helper function as they are duplicated in at least 3 places

Reply:
@AKalinich-Luxoft  hmi_notifications_test.cc contains a lot of other tests, not related to the task. Most of them have different basic expectations. Helper function would be to specific to all this variety.